geometry_msgs/Point point
string color